mailpoet/assets/js/src/date.ts

```diff
@@ -178,5 +178,5 @@ export const MailPoetDate : {
 
     return convertedFormat.join('');
   },
-  isInFuture: (dateString: string): boolean => new Date(dateString).getTime() > Date.now(),
+  isInFuture: (dateString: string, currentTime: MomentInput): boolean => Moment(dateString).isAfter(currentTime, 's'),
```

Comment:
So I guess we don't use this function in any other place?
Seems like `currentTime` could be optional. But if we don't use it elsewhere, maybe not too necessary.